src/hooks/usePlaygroundRendererCode.ts

```diff
@@ -0,0 +1,19 @@
+import { Code } from "@/types";
+import { DEFAULT_ADDON_STATE, PLAYGROUND_EDITOR_CHANGED } from "@/consts";
+import { useCallback, useState, useChannel } from "@storybook/preview-api";
+
+const usePlaygroundRendererCode = () => {
```

Comment:
```suggestion
const usePlaygroundCodeRenderer = () => {
```

Reply:
I called everything that's related to the rendering part "PlaygroundRenderer" 😅 I wanted to leave the same terminology
If we make a change I think we should change the name for all the rendering related